.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,7 +1,7 @@
 contact_links:
-  - name: Ask a question during our office hours
-    url: https://calendly.com/fivetran-solutions-team/fivetran-solutions-team-office-hours
-    about: Schedule time during the external office hours block with the Fivetran Analytics Engineering team for support
+  - name: Ask a question during our office hours 
+    url: https://calendly.com/fivetran-solutions-team/fivetran-solutions-team-office-hours 
+    about: Schedule time during the external office hours block with the Fivetran Analytics Engineering team for support 
```

Comment:
Similar to the comment about removing the direct link to the office hours, can we update this section to remove the office hours link and resemble the [format we have in Zendesk](https://github.com/fivetran/dbt_zendesk/blob/main/.github/ISSUE_TEMPLATE/config.yml) to direct users to the community

Reply:
updated